src/cmd-buildupload

```diff
@@ -109,6 +117,12 @@ def s3_upload_build(args, builddir, bucket, prefix):
             s3_path = f'{prefix}/{nogz}'
             set_content_disposition = True
 
+        # Mark artifacts that we don't want to upload as already uploaded
+        # so they'll get ignored later.
+        if imgname not in upload_artifacts:
+            uploaded.add(bn)
```

Comment:
Could we print something like:

> Skipping artifact {imgname}

here?

Reply:
done